Source/missiles.cpp

```diff
@@ -2812,15 +2812,16 @@ void MI_LArrow(Missile &missile)
 {
 	missile._mirange--;
 	int p = missile._misource;
+	auto &player = Players[p];

```

Comment:
In this case you want to put the declaration of `player` inside each if statement where it gets used. If _misource is -1 this creates a reference to an invalid area of memory currently.